AK/Concepts.h

```diff
@@ -111,6 +111,12 @@ concept IterableContainer = requires
     { declval<T>().begin() } -> IteratorPairWith<decltype(declval<T>().end())>;
 };
 
+template<typename T, typename InnerType = typename T::ValueType>
+inline constexpr bool IsAppendableContainer = requires(T container, InnerType element)
```

Comment:
This is not a concept; if you're not adding a concept then why do you say so in the commit message?